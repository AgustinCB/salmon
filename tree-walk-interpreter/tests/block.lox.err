There was an error! [line 8] Error: Variable `a` not initialized!
