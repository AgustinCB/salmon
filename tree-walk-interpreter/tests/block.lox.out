BEFORE
ON THE BLOCK
ASDASD
